charts/controller/crds/deployments.plural.sh_servicedeployments.yaml

```diff
@@ -73,6 +73,9 @@ spec:
                       type: object
                     type: array
                 type: object
+                x-kubernetes-validations:
```

Comment:
Bindings should be mutable, I think. It won't happen *that* often but it will occasionally.

Reply:
It is possible in general, but we do not have any logic to update bindings on change. It would require running a different `updateRbac` mutation. We would also need to keep separate SHA just for bindings to check if they have changed to update them independently of i.e. Service/Cluster.

I can add an issue to handle RBAC updates and work on that next. Right now any change to bindings would not be reflected in the system. They are only used during initial creation.

Reply:
Ok fair enough.  It's also possible I can add that back to the main mutation, since it's going to be awkward to manage separately everywhere.

Reply:
Ye, that would be the best if we could also pass bindings to update mutation for cluster and service. This way it will be pretty much automatically handled by the controller. Only small change to update attributes will be needed.

Reply:
Let's just configure them as mutable and I'll make the console api change in the next release and we should be good.